Tests/CarthageKitTests/SimulatorSpec.swift

```diff
@@ -146,6 +146,26 @@ class SimulatorSpec: QuickSpec {
 					}
 				}
 
+				context("Xcode 10.1 after installing 10.2 beta") {
+					it("should return the first simulator of the latest version") {
+						let data = loadJSON(for: "Simulators/availables-xcode101-after-installing-102-beta")
```

Comment:
The same holds for this fixture name.

Like `with-unavailable-newer-simulators`

Sorry, I have no better idea. 😅

Reply:
I think your idea is much better than mine was :)